### ISSUE TITLE
Videomass on Ubuntu Bionic ?

### ISSUE THREAD

**Issue by tissatussa** [jeanslack/Videomass#18]
i only find a PPA for Ubuntu 20.xx .. do you support Ubuntu Bionic 18.04 ?  Or should i compile & install Videomass myself for this Ubuntu version ?

**Comment by jeanslack** [comment]
Unfortunately, there is no PPA support for Ubuntu Bionic, nor for previous versions. Some dependencies like python3-pubsub are not available on the official Ubuntu 18.04 repos and not even on older versions. 
You'll have to do something manually using pip.

**Comment by arrowgent** [comment]
this is a bit late but you can install the focal python3-pubsub to avoid the dependencies
https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=python3-pubsub&searchon=names

i doubt there are any significant changes, just that the maintainer has not included it to bionic

this comes with the caveat that possibly pubsub doesnt work or will break videomass or incomplete python3 compatibility.


however, ideally your PPA could include the bionic package for python3-pubsub.  you'd have to maintain atleast the minimum version and any security backports (if you desired)

**Comment by jeanslack** [comment]
@arrowgent 

Thanks for the info.
When this issue was opened the Videomass package for Ubuntu Bionic was not yet included in the PPA, it was only for Focal. Some time later, I had decided to support other Ubuntu versions as well including Bionic, but the python3-pubsub package on Bionic has to be installed manually using pip or by installing the Ubuntu Focal one as you suggested.

Including the bionic python3-pubsub package on my PPA is another possibility I might consider. I'll see how.

Also, I have decided to keep this issue open so that other Ubuntu Bionic users can easily see how to fix the problem.
